### PR TITLE
Kyra Bergsund

### PR DESCRIPTION
**Pull request by kbergsund**
Submitting my Mod 1 Pre-Work.

### CODE REVIEW

File: finalPrep/README.md [hunk 1]
```diff
@@ -18,37 +18,41 @@ Some things to consider as you refactor include...
...
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+`This will definitely change when I have a better understanding of standard weekly group work. I will be able to get more granular with homework v. project v. study time. And I will build in time to meet with collaborators based on everyone's availability.`
+
+- `Add Week 1 Screenshot Here`![](assets/README-826b900a.png)
+
```
Comment (francepack) [inline finalPrep/README.md:37]: Great stuff here! Looks like you have a routine set up, including time planned for weekend work. Things inevitably change, so I am glad to see the Saturday morning reflections to adjust for the upcoming week.
- I try to take stock of what went well, and what didn't go so well to try to be realistic about how I allot time in the upcoming week.
- When I know what I need to work on, I will edit calendar events to bring more focus into the task at hand for that block. Sometimes, a 2 hour homework block would be unfocussed for me, so I would pick a thing, maybe 2, for that time.

File: finalPrep/README.md [hunk 2]
```diff
@@ -18,37 +18,41 @@ Some things to consider as you refactor include...
...
+- `Add Week 2 Screenshot Here`![](assets/README-115c9eef.png)
+
+- `Add Week 3 Screenshot Here`![](assets/README-440d1e3b.png)
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist [HERE](https://gist.github.com/kbergsund/446dbb723a3560a61bfcaa51a94e9d39):
```
Comment (francepack) [inline finalPrep/README.md:45]: Good job here. In your intro letter, you could think about adding a sentence about what you are excited/nervous about as you start Turing.

File: finalPrep/README.md [hunk 3]
```diff
@@ -18,37 +18,41 @@ Some things to consider as you refactor include...
...
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [x] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [x] Add a link to your gist [HERE](https://gist.github.com/kbergsund/deb2c028794c20802a6049bb49731ae8)
```
Comment (francepack) [inline finalPrep/README.md:50]: Excellent notes! I really like how you can scan the left side for a quick glance at key content/terms, as well as using some different color.

One tip- no need to be shy about leaving open lines, etc. You never know when you may want to fill in a bit more on a topic later.

File: finalPrep/README.md [hunk 4]
```diff
@@ -18,37 +18,41 @@ Some things to consider as you refactor include...
...
 
 ### Group Work Prep
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [x] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [x] Add a link to your gist [HERE](https://gist.github.com/kbergsund/1bf79941d0092f8f9418e83df7f2a5f3)
```
Comment (francepack) [inline finalPrep/README.md:55]: Excellent thoughts here- It is great you are self-aware about some work/life balance struggles associated with sharing a work space with a partner- you'll find coding takes pretty focussed energy at times, so glad you are thinking about how to work with your partner through this.

File: finalPrep/annotations.js [hunk 5]
```diff
@@ -2,41 +2,69 @@
...
 function buildABear(name, age, fur, clothes, specialPower) {
+  // Declares new variable within function that is a string. It interpolates the name parameter so that when the function is called, the name argument will be 'filled into' the string.
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+  // Declares new variable within function that is an array combining the name and age parameters.
   var demographics = [name, age];
+  // Declares new variable within function that is a string. It concatenates the specialPower parameter so that when the function is called, the specialPower argument will be 'filled into' the string.
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+  // Declares a new object variable within function. The object has six different object properties defined in key-value pairs.
   var builtBear = {
+    // The basicInfo key has the value of the above demographics variable.
```
Comment (francepack) [inline finalPrep/annotations.js:16]: Nice job distinguishing where your values come from.

File: finalPrep/annotations.js [hunk 6]
```diff
@@ -2,41 +2,69 @@
...
   for (var i = 0; i <= range; i++) {
+    // Sets up a conditional statement that checks two conditions. The && sign means that both conditions must be true in order for this first condition to be met. Condition #1 evaluates if the modulo, or remainder, of our counter variable `i` divided by the num1 parameter is strictly equal to 0. Condition #2 evaluates whether the same is true but for the modulo of `i` divided by the num2 parameter. Only if both modulos are 0 does the code in the curly bracket execute at which time the computer also exits the conditional statement and does not evaluate any below conditions. Otherwise, if not met, it moves on to the next condition.
```
Comment (francepack) [inline finalPrep/annotations.js:47]: Nice explanations. You have clearly digested what a modulo is, as well as comparative operators. Love to see the practice with technical explaination.

File: finalPrep/modZeroHero.js [hunk 7]
```diff
@@ -1,52 +1,88 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
+var heroName = "Kyra";
```
Comment (francepack) [inline finalPrep/modZeroHero.js:4]: I figured it out- I worked with a Kylie, but what threw me off is my wife's niece is named Kyra (eye, not key)!

File: finalPrep/modZeroHero.js [hunk 8]
```diff
@@ -1,52 +1,88 @@
...
-//   - saveTheDay should be a string a hero would say once they save the day 
+//   - saveTheDay should be a string a hero would say once they save the day
 //   - badExcuse should be a string a hero would say if they are too afraid of the dangerLevel
-
-// Your function should include an if/else statement that meets the following criteria
-//   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
-//   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
-//   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
+function assessSituation(dangerLevel, saveTheDay, badExcuse) {
+  dangerLevel = dangerLevel;
```
Comment (francepack) [inline finalPrep/modZeroHero.js:56]: You shouldn't need these reassignments- you should be able to use the params as written in the parens.

File: finalPrep/modZeroHero.js [hunk 9]
```diff
@@ -1,52 +1,88 @@
...
+  badExcuse = badExcuse;
+
+  // Your function should include an if/else statement that meets the following criteria
+  //   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the badExcuse to the console
+  //   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
+  //   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
+
+  if (dangerLevel > 50) {
+    console.log(badExcuse);
+  } else if (dangerLevel > 10) {
```
Comment (francepack) [inline finalPrep/modZeroHero.js:67]: This works, but could be a little 'leaky' if things changed. You may think about saving the complex logic for your `else` block, and take care of the upper and lower limits for your if/elseif blocks.

File: finalPrep/modZeroHero.js [hunk 10]
```diff
@@ -56,29 +92,80 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
...
+  sayName() {
+    console.log(this.name);
+  }
+
+  maximizeEnergy() {
+    this.energyLevel = 1000;
+  }
+
+  gainPower(increaseBy) {
+    this.powerLevel += increaseBy;
```
Comment (francepack) [inline finalPrep/modZeroHero.js:143]: Nice dynamic function!

File: section4/exercises/classMethods.js [hunk 11]
```diff
@@ -7,34 +7,63 @@ Complete each task, using the Burrito class below as a starting point.
...
 
   // ADD CODE
+
+  // Task 3: changeProtein method
+  changeProtein(newProtein) {
+    this.protein = newProtein;
+  }
+
+  // Task 5: addTopping method
+  addTopping(newTopping) {
```
Comment (francepack) [inline section4/exercises/classMethods.js:46]: Nice job using params in different ways to achieve the desired results!

**Review by francepack — COMMENTED** [review]
@kbergsund  I just had a chance to go over your prework and you are officially `turing ready`! I added some comments to a few sections for review. Keep the coding momentum going as you prepare for Mod1, and I suggest taking a look at some of the first parts of curriculum [here](https://frontend.turing.edu/).

Let me know if you have any questions about Mod0 material, or life in general at Turing- excited for you to start!

File: finalPrep/README.md (hunk 1 above)
Comment (kbergsund) [reply finalPrep/README.md:37]: Thanks so much for your thorough feedback and support. Just reviewed in detail after a busy couple of  weeks at work. Duly noted all your suggestions and tips!

**Pull request closed without merging**